db/reporting_structure.sql

```diff
@@ -15,50 +15,50 @@ SET row_security = off;
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::Parents' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_family_parents VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::ParentingYouth' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_parenting_youth VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::ParentingChildren' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_parenting_children VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::UnaccompaniedMinors' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_unaccompanied_minors VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::IndividualAdults' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_individual_adults VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::NonVeteran' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_non_veteran VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::Family' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_family VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::YouthFamilies' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_youth_families VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'Reporting::MonthlyReports::Children' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_children VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'AdultOnlyHouseholdsSubPop::Reporting::MonthlyReports::AdultOnlyHouseholds' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_adult_only_households VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'AdultsWithChildrenSubPop::Reporting::MonthlyReports::AdultsWithChildren' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_adults_with_children VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'ChildOnlyHouseholdsSubPop::Reporting::MonthlyReports::ChildOnlyHouseholds' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_child_only_households VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'ClientsSubPop::Reporting::MonthlyReports::Clients' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_clients VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'NonVeteransSubPop::Reporting::MonthlyReports::NonVeterans' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_non_veterans VALUES (NEW.*);
-           ELSIF  ( NEW.type = 'VeteransSubPop::Reporting::MonthlyReports::Veterans' ) THEN
-              INSERT INTO warehouse_partitioned_monthly_reports_veterans VALUES (NEW.*);
-          
-        ELSE
-          INSERT INTO warehouse_partitioned_monthly_reports_unknown VALUES (NEW.*);
-          END IF;
-          RETURN NULL;
-      END;
+    AS $$

+        BEGIN


```

Comment:
Something odd seems to be happening here.  Pretty sure this file shouldn't have changed anyway.

Reply:
This happens when you open the file in vscode and save it.

Reply:
@eanders Any tips on how to revert this? I think I need to go back to that commit and remove this file from it, because `git checkout` isn't working. I even copied the entire file from `release-68` and replaced it with that, but it still wants to apply these changes. Any advice would be appreciated

Reply:
I have noticed issues with that as well, github doesn't recognize the carriage return change as a change. I think it has worked for me to delete the file and then run `git checkout release-68 db/reporting_structure.sql`

Reply:
I tried both of those things and neither seemed to take

Reply:
Alternatively, just change the line endings in VSCode...